libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -299,7 +299,12 @@ void AP_BoardConfig::px4_setup_drivers(void)
 
     if (px4.board_type == PX4_BOARD_PH2SLIM ||
         px4.board_type == PX4_BOARD_PIXHAWK2) {
-        _imu_target_temperature.set_default(60);
+        _imu_target_temperature.set_default(45);
```

Comment:
I still want it as 60, and anyone who is serious about the best results from their system should still set it to 60...

However, it can take 15 minutes to get to 60, so for the RTF user, or the weekend flyer, 45 is an acceptable compromise, and is still far better than having no heater.